Data/Settings/Lists.txt

```diff
@@ -25,7 +25,7 @@ END
 SEL
 m_Visible:1
 m_X:0
-m_Y:82
+m_Y:81
```

Comment:
Revert position changes to avoid conflicts with with custom user preferences when updating with installer

Reply:
I should have discarded that change. nvm